haystack/core/pipeline/base.py

```diff
@@ -766,7 +766,7 @@ def _prepare_component_input_data(self, data: Dict[str, Any]) -> Dict[str, Dict[
 
         return data
 
-    def _init_inputs_state(self, data: Dict[str, Dict[str, Any]]) -> Dict[str, Dict[str, Any]]:
+    def _normalize_input_data(self, data: Dict[str, Dict[str, Any]]) -> Dict[str, Dict[str, Any]]:
```

Comment:
A comment that briefly explains what this function does (as "normalization" is somewhat generic).